catalog-rest-service/src/main/java/org/openmetadata/catalog/atlas/AtlasPublisherConfiguration.java

```diff
@@ -0,0 +1,11 @@
+package org.openmetadata.catalog.atlas;
+
+import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
+import org.openmetadata.catalog.type.EventFilter;
+
+public class AtlasPublisherConfiguration {
```

Comment:
I think it is possible to set 
@Getter and @Setter on the class itself and it applies to all the fields.. 

Haven't tried this myself